search/src/main/scala/weco/api/search/services/ImagesRequestBuilder.scala

```diff
@@ -141,7 +147,30 @@ class ImagesRequestBuilder(queryConfig: QueryConfig)
     minScore: Double
   ): SearchRequest =
     search(index)
-      .query(query(imageId, image, index))
       .size(n)
       .minScore(minScore)
+      .query(query(imageId, image, index))
+
+  private def rawSimilarityRequest(
+    query: (String, IndexedImage, Index) => JsonObject
+  )(
+    index: Index,
+    imageId: String,
+    image: IndexedImage,
+    n: Int,
+    minScore: Double
+  ): SearchRequest =
+    search(index).source(
+      Json
+        .fromJsonObject(
+          query(imageId, image, index)
+            .add("min_score", Json.fromDouble(minScore).get)
+            // n is the number of similar documents we want the query to return
+            // However, because this is used by KNN searches, which also return
+            // the document in question (which is later filtered out of the result list),
+            // an extra slot must be added for the source document to occupy.
+            .add("size", Json.fromInt(n + 1))
```

Comment:
Do we still need to fetch `n + 1` images if we're excluding the original image at source?